storage/trident-iscsi-tools/pkg.yaml

```diff
@@ -0,0 +1,46 @@
+name: trident-iscsi-tools
+variant: scratch
+shell: /bin/bash
+dependencies:
+  - stage: base
+steps:
+  - sources:
+      - url: "https://sg.danny.cz/scsi/lsscsi-{{ .LSSCSI_VERSION }}.tar.gz"
+        destination: lsscsi.tar.gz
+        sha256: {{ .LSSCSI_SHA256 }}
+        sha512: {{ .LSSCSI_SHA512 }}
+    prepare:
+      - |
+        sed -i 's#$VERSION#{{ .VERSION }}#' /pkg/manifest.yaml
+      - |
+        tar -xzf lsscsi.tar.gz --strip-components=1
+    build:
+      - |
+        ./configure --prefix=/usr/local --exec-prefix=/usr/local
+        # make -j $(nproc) DESTDIR=/rootfs
+    install:
+      - |
+        make DESTDIR=/rootfs install
+      # Remove man pages and info pages to save space
+      - |
+        rm -rf /rootfs/usr/local/share
+      - |
+        mkdir -p /rootfs/usr/local/bin
+        mkdir -p /rootfs/usr/local/sbin
+        cp /usr/bin/ls /rootfs/usr/local/bin
```

Comment:
I don't like this, copying over basic tools into talos when all these can be done via code, then it makes for this to be a separate package (https://github.com/siderolabs/extensions/pull/882/files#r2525625614 ignore this comment)

Reply:
since it's on the host, you can copy from `base` since it's all built from same toolchain